cms/bundles/mixins.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
```

Comment:
This had to be moved to a separate file to avoid a circular dependency.

`Bundle` uses the `FutureReleaseCalendarChooserWidget`, which depends on `ReleaseCalendarPage` which in turn uses the `BundledPageMixin` (and the mixin uses `Bundle`, closing the circle).